src/screens/help.test.ts

```diff
@@ -169,4 +169,34 @@ describe('#helpScreen()', () => {
 			'What you want to list (required) (accepts: toppings, crusts, two words)'
 		);
 	});
+
+	it('Emits help with dynamic async accepts properties', async () => {
+		process.argv = [
+			'/path/to/node',
+			path.join(testProjectsPath, 'pizza-ordering', 'cli', 'entry.js'),
+			'order',
+			'dynamic-async-accepts',
+			'--help',
+		];
+
+		const result = removeFormatting(await helpScreen());
+
+		expect(result).toContain('What type of pizza to order (accepts: a, b, c, d)');
+		expect(result).toContain('Just used for testing (accepts: a1, b2, c3, d4)');
+	});
+
+	// it('Emits help with dynamic sync accepts properties', async () => {
+	// 	process.argv = [
+	// 		'/path/to/node',
+	// 		path.join(testProjectsPath, 'pizza-ordering', 'cli', 'entry.js'),
+	// 		'order',
+	// 		'dynamic-sync-accepts',
+	// 		'--help',
+	// 	];
+
+	// 	const result = removeFormatting(await helpScreen());
+
+	// 	expect(result).toContain('What type of pizza to order (accepts: a, b, c, d)');
+	// 	expect(result).toContain('Just used for testing (accepts: a1, b2, c3, d4)');
+	// });
```

Comment:
Why is this test commented-out?

Reply:
This was commented out while I am looking into understanding why Codecov thinks there is a gap in coverage.  I think it too has issues with checking the data-providing callback `constructor.name`, but it's very weird, because the same test is done earlier in code, and it's not complaining about that.